pipeline_0_annotate.sh

```diff
@@ -61,3 +63,104 @@ vep \
 --fork 4
 
 module unload VEP
+
+#CAPICE
+
+CAPICE_INPUT="${INPUT}"
+CAPICE_OUTPUT_DIR="${ANNOTATE_OUTPUT_DIR}"/step0b_capice
+
+if [[ ${OUTPUT_FILE} == *vcf ]]
+then
+    CAPICE_OUTPUT="${CAPICE_OUTPUT_DIR}"/"${OUTPUT_FILE/.vcf/.tsv}"
+    CAPICE_OUTPUT_VCF="${CAPICE_OUTPUT_DIR}"/"${OUTPUT_FILE/.vcf/.vcf.gz}"
+    CAPICE_OUTPUT_LOG="${CAPICE_OUTPUT}.log"
+else
+    CAPICE_OUTPUT="${CAPICE_OUTPUT_DIR}"/"${OUTPUT_FILE/.vcf.gz/.tsv}"
+    CAPICE_OUTPUT_VCF="${CAPICE_OUTPUT_DIR}"/"${OUTPUT_FILE}"
+    CAPICE_OUTPUT_LOG="${CAPICE_OUTPUT/.tsv/.log}"
+fi
+
+mkdir -p "${CAPICE_OUTPUT_DIR}"
+
+if [ -f "$CAPICE_OUTPUT.vcf.gz" ]
+then
+        if [ "$FORCE" == "1" ]
+        then
+                rm "$CAPICE_OUTPUT.vcf.gz"
+        else
+                echo "$CAPICE_OUTPUT.vcf.gz already exists, use -f to overwrite.
+                "
+                exit 2
+        fi
+fi
+
+if [ -f "$CAPICE_OUTPUT_LOG" ]
+then
+        if [ "$FORCE" == "1" ]
+        then
+                rm "$CAPICE_OUTPUT_LOG"
+        else
+                echo "$CAPICE_OUTPUT_LOG already exists, use -f to overwrite.
+                "
+                exit 2
+        fi
+fi
+
+module load CADD
+# strip headers from input vcf for cadd
+gunzip -c $CAPICE_INPUT | sed '/^#/d' | bgzip > ${CAPICE_OUTPUT_DIR}/input_headerless.vcf.gz
+CADD.sh -a -g GRCh37 -o ${CAPICE_OUTPUT_DIR}/cadd.tsv.gz ${CAPICE_OUTPUT_DIR}/input_headerless.vcf.gz
+module unload CADD
+
+module load CAPICE
+python ${EBROOTCAPICE}/CAPICE_scripts/model_inference.py \
+--input_path ${CAPICE_OUTPUT_DIR}/cadd.tsv.gz \
+--model_path ${EBROOTCAPICE}/CAPICE_model/xgb_booster.pickle.dat \
+--prediction_savepath ${CAPICE_OUTPUT} \
+--log_path ${CAPICE_OUTPUT_LOG} &> ${CAPICE_OUTPUT_LOG}
```

Comment:
What is our logging strategy for the pipeline? Log everything to ```STDOUT``` and ```STDERR``` and let the user decide whether they want to redirect it to file(s). Redirect to separate files for each tool? Maintain multiple logs?